lib/bs/world/factory.ex

```diff
@@ -88,7 +88,7 @@ defmodule Bs.World.Factory do
 
     update_in(world.snakes, fn snakes ->
       for snake <- snakes do
-        {:ok, point} = World.rand_unoccupied_space(world)
+        {:ok, point} = World.rand_unoccupied_space(world, 1, true)
```

Comment:
Do you want to randomly set `only_odd_squares`? This would allow spawning on even spaces. It doesn't _really_ matter since it's basically equivalent, but might _look_ weird to never see spawns on the even ones (or odd?).

Reply:
that seems like a good improvement. I would add it on world.ex:66 random choice of 0 or 1 outside of the for loops.

Reply:
@cbinners I added that functionality to this pr, if you have thoughts?

Reply:
@joram I was thinking you could generate the boolean value after line 89: https://github.com/sendwithus/battlesnake-server/blob/fd84031b56a044c15bce20b9c7e19a9d4add2703/lib/bs/world/factory.ex#L89

something like:

```elixir
use_odd_spacing = Enum.random(0..1) == 0
{:ok, point} = World.rand_unoccupied_space(world, 1, use_odd_spacing)
```

Thoughts?